dotcom-rendering/src/components/Card/components/UL.tsx

```diff
@@ -11,11 +11,22 @@ const ulStyles = (direction: Direction) => css`
 	position: relative;
 	display: flex;
 	flex-direction: ${direction};
-	row-gap: 12px;
 	${until.tablet} {
 		flex-direction: column;
 		width: 100%;
 	}
+
+	& > li {
+		margin-bottom: ${space[3]}px;
+	}
+
+	@supports (row-gap: 1em) {
+		& > li {
+			margin-bottom: 0;
+		}
+		/* Supported in flex layout is lacking: https://developer.mozilla.org/en-US/docs/Web/CSS/row-gap#browser_compatibility */
```

Comment:
Are we planning on keeping both implementations? Assuming not, should we put this into a method that makes it easy to remove later?

[We use this in a few places](https://github.com/search?q=repo%3Aguardian%2Fdotcom-rendering%20row-gap&type=code) - and I assume we would have similar issues here.

e.g. (not a great solution, but more trying to focus on the "how do we remove these backwards compatibility things")
```ts
const rowGap(selector: string, px: number) => css`
  ${selector} {
    margin-bottom: ${px}px;
  }
  
  @supports (row-gap: 1em) {
    ${selector} {
      margin-bottom: 0;
    }
    row-gap: ${px}px;
`
```